core/src/main/kotlin/model/Documentable.kt

```diff
@@ -332,8 +332,9 @@ data class DTypeAlias(
     override val documentation: SourceSetDependent<DocumentationNode>,
     override val expectPresentInSet: DokkaSourceSet?,
     override val sourceSets: Set<DokkaSourceSet>,
-    override val extra: PropertyContainer<DTypeAlias> = PropertyContainer.empty()
-) : Documentable(), WithType, WithVisibility, WithExtraProperties<DTypeAlias> {
+    override val extra: PropertyContainer<DTypeAlias> = PropertyContainer.empty(),
+    override val generics: List<DTypeParameter>
```

Comment:
@Kordyjan Do you agree with this solution? The rest of the PR looks good to me ☺️

Reply:
Yup, we've forgotten that typealiase can have generic parameters, so it definitely should be here.
Nitpick: by convention and since `generics` is a parameter without default value it should go before the `extra`.

Reply:
Updated